packages/engine/src/worker/runner/python/mod.rs

```diff
@@ -256,3 +257,36 @@ async fn _run(
     // }
     Ok(())
 }
+
+pub(crate) fn cleanup_python_runner(experiment_id: &ExperimentId, exit_status: EngineExitStatus) {
+    // Cleanup python socket files in case the engine didn't
+    let frompy_files = glob::glob(&format!("{experiment_id}-frompy*"));
+    let topy_files = glob::glob(&format!("{experiment_id}-topy*"));
+
+    // We're ignoring glob errors as they shouldn't stop the whole cleanup process.
+    frompy_files
+        .into_iter()
+        .chain(topy_files)
+        .flatten()
+        .filter_map(Result::ok)
+        .for_each(|path| match std::fs::remove_file(&path) {
+            Ok(_) => {
+                match exit_status {
+                    EngineExitStatus::Success => tracing::error!(
+                        experiment = %experiment_id,
+                        "Removed file {path:?} that should've been cleanup by the engine."
+                    ),
+                    EngineExitStatus::Error => tracing::warn!(
+                        experiment = %experiment_id,
+                        "Removed file {path:?} that should've been cleanup by the engine."
+                    ),
```

Comment:
What is the point of this match at all

Reply:
Removed `EngineExitStatus` in https://github.com/hashintel/hash/pull/494/commits/ce7c3a4f39d8e261bc0ff35dbbd01471becbc471